cmd/bblfshctl/cmd/driver_remove.go

```diff
@@ -48,12 +48,13 @@ func (c *DriverRemoveCommand) Execute(args []string) error {
 
 	for _, lang := range langs {
 		r, err := c.srv.RemoveDriver(ctx, &protocol.RemoveDriverRequest{Language: lang})
-		if err != nil || len(r.Errors) > 0 {
+		if err != nil {
+			return err
```

Comment:
Alternative: check for `r != nil` before this block and print the errors, then exit (or not, if `err == nil`). This should cover the case where `err != nil` and `r` contains informative errors (which I don't know if it's the case with the current implementation of RemoveDriver, but could be in the future).

Reply:
`RemoveDriver` is from the auto-generated client, so it won't ever return both non-nil error and non-nil response.

Reply:
I wonder if there is a way to express this kind of invariant in the code 🤔 

As I do not see a way for language to support this, obvious one would be to have a test..